en/physics/physics-event.md

```diff
@@ -75,11 +75,11 @@ Collision events are divided into three types:
 
 Where the collision pairs that can generate collision events are:
 
-| Type | Static rigid body | Kinematic rigid body | Dynamic rigid body |
+| Type | **Static rigid body** | **Kinematic rigid body** | **Dynamic rigid body** |
```

Comment:
I'm sorry. I was not clear. You don't need to bold the table headings, just the key of each row.

Change this back to: 

| Type | Static rigid body | Kinematic rigid body | Dynamic rigid body |